packages/deploy-script-support/src/writeCoreProposal.js

```diff
@@ -150,10 +150,11 @@ export const makeWriteCoreProposal = (
 
 const manifestBundleRef = ${stringify(manifestBundleRef)};
 const getManifestCall = harden(${stringify(getManifestCall, true)});
-const overrideManifest = ${stringify(overrideManifest, true)};
+const customManifest = ${stringify(customManifest, true)};
 
-// Make the behavior the completion value.
-(${makeCoreProposalBehavior})({ manifestBundleRef, getManifestCall, overrideManifest, E });
+// Make a behavior function and "export" it by way of script completion value.
+const behavior = (${makeCoreProposalBehavior})({ manifestBundleRef, getManifestCall, customManifest, E });
+behavior;
```

Comment:
Consider this for clarity, as well:

```suggestion
const makeCoreProposalBehavior = ${makeCoreProposalBehavior};

// Make a behavior function and "export" it by way of script completion value.
const behavior = makeCoreProposalBehavior({ manifestBundleRef, getManifestCall, customManifest, E });
behavior;
```